commands/serve/lib/serve.js

```diff
@@ -138,6 +138,7 @@ module.exports = async (argv) => {
         watch: true,
         config: serveConfig.config,
         cwd: context,
+        replacementStrings: serveConfig.replacementStrings,
```

Comment:
when investagating why this was added I think I found that the config path is not passed correcly to build.
(I have not found why yet)

Reply:
I know, and I couldn't figure it out, that's why I solved it like this...

Reply:
it can be fixed by changing
```
defaultBuildConfig = initConfig(yargs([])).argv;
```
to
```
const yargsArgs = argv.config ? ['--config', argv.config] : [];
defaultBuildConfig = initConfig(yargs(yargsArgs)).argv;
```
in [commands/build/lib/build.js](https://github.com/qlik-oss/nebula.js/blob/57a80910072d79d8e501f9b0b685495b24a87178/commands/build/lib/build.js#L219)

an alternative would be to not use yargs to load the config file (that would also make it possible for replacementStrings to be a normal object in the config file)

Reply:
Thanks for the feedback! I will look into loading the config file in another way, not using yargs...

Reply:
I did what you suggested to solve the problem with the config path.
I also changed the format of `replacementStrings` into an object. I had to set the rule `dot-notations` to `false` through `yargs.parserConfiguration` to get it working nicely though.